aioxmpp/__init__.py

```diff
@@ -134,6 +134,7 @@
 from .pep import PEPClient  # NOQA
 from .bookmarks import BookmarkClient  # NOQA
 from .version import VersionServer  # NOQA
+from .mdr import DeliveryReceiptsService # NOQA
```

Comment:
PEP8 violation (there need to be two spaces before `#`).

Reply:
good catch